model_compression_toolkit/pytorch/mixed_precision/mixed_precision_wrapper.py

```diff
@@ -0,0 +1,149 @@
+# Copyright 2022 Sony Semiconductors Israel, Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
```

Comment:
I couldn't write it in the right place, cause the file is empty, but please add license to https://github.com/ofirgo/model_optimization/blob/ofirgo/pytorch-mp/model_compression_toolkit/pytorch/mixed_precision/__init__.py